.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "prettier.jsxBracketSameLine": true,
+  "prettier.jsxSingleQuote": true,
+  "prettier.singleQuote": true
+}
```

Comment:
Is this agreed upon by all team members? The reason I ask is that I often see different standards in your PR's

Reply:
yes We just talked it over. The workspace settings should change the settings for everyone writing in the project.